src/commons/utils/corio_utils.py

```diff
@@ -114,4 +117,19 @@ def run_local_cmd(cmd: str) -> tuple:
     finally:
         if proc:
             proc.terminate()
-            
\ No newline at end of file
+
+
+def create_file(path: str, size: int):
+    """
+    Create file with random data
+
+    :param size: Size in bytes
+    :param path: File name with path
+    """
+    base = 1024 * 1024
+    while size > base:
+        with open(path, 'ab+') as f_out:
+            f_out.write(os.urandom(base))
```

Comment:
I see ur using os.urandom again will this fix memory consumption issue? Have you tested this?

Reply:
This will reduce memory consumption, since earlier we were generating large objects in one go. Now we are generating large random data in chunks. I have tested this.